Sample/OCKSample/SampleData.swift

```diff
@@ -59,65 +59,65 @@ class SampleData: NSObject {
             emailAddress: "mruiz2@mac.com",
             monogram: "MR",
             image: nil),
-        
+
         OCKContact(contactType: .CareTeam,
             name: "Bill James",
             relation: "Nurse",
             tintColor: Colors.Green.color,
             phoneNumber: CNPhoneNumber(stringValue: "888-555-5512"),
             messageNumber: CNPhoneNumber(stringValue: "888-555-5512"),
             emailAddress: "billjames2@mac.com",
-            monogram: "BJ",
+            monogram: nil,
             image: nil),
-        
+
         OCKContact(contactType: .Personal,
-            name: "Tom Clark",
+            name: "Tom Van de Clark",
```

Comment:
I put that only to check the issue w/ multiple words inside an user name, but we can for sure remove it.